### ISSUE TITLE
Black screen on launch

### ISSUE THREAD

**Issue by tyrone-sudeium** [tyrone-sudeium/LaunchXIV#3]
Shadowbringers + Catalina issue. When launching the game, the FINAL FANTASY XIV window appears, some random process (perl? Terminal? wineloader?) will request "full keyboard access", and the game will stall in a black window.

I suspect the problem is mostly because of Catalina -- because the game is being launched via `NSWorkspace`, it is not inheriting either `FINAL FANTASY XIV ONLINE.app`'s permissions, or `LaunchXIV.app`'s for that matter. Exactly what permissions it gets is a bit nebulous. So I think wine stalls because there's some low level syscall it's doing that just gets held up by the macOS security system.

It should be possible to use a lower level `exec`-style function to start the game, possibly requiring it to run as a child of `LaunchXIV`, which wouldn't be ideal since `LaunchXIV` would stay running, but it beats a black screen.
